content/en/docs/spinnaker/policy-engine.md

```diff
@@ -52,7 +52,7 @@ spec:
 
 *Note: There must be a trailing /v1 on the URL. This extension is only compatible with OPA's v1 API.*
 
-If you are using an in-cluster OPA instance (such as one set up with the instructions below), Spinnaker can access OPA via the Kubernetes service DNS name. The following example configures Spinnaker to connect with an OPA server at http://opa.opaserver:8181:
+If you are using an in-cluster OPA instance (such as one set up with the instructions below), Spinnaker can access OPA via the Kubernetes service DNS name. The following example configures Spinnaker to connect with an OPA server at http://opa.opa:8181:
 
```

Comment:
Nit: should this be in backticks like it is on L103? `http://opa.opa:8181` (also wonder how it got past the link checker)